packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.vue

```diff
@@ -0,0 +1,93 @@
+<script lang="ts" setup>
+import type { OnyxIconProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxIconProps>(), {
+  size: "sm",
+  color: "currentColor",
+});
+</script>
+
+<template>
+  <!-- eslint-disable vue/no-v-html -->
+  <i
+    class="onyx-icon"
+    :class="[
+      props.size !== 'sm' ? `onyx-icon--${props.size}` : '',
+      props.color !== 'currentColor' ? `onyx-icon--${props.color}` : '',
+    ]"
+    aria-hidden="true"
```

Comment:
is aria-hidden a good idea? shouldne we rather offer an alt with e.g. 
"Icon of ${iconName}" that can be overwritten or so?

-> Image you have an icon button, then as a screen reader user you would just get an info that it is a button but no idea what it is about?

Reply:
I asked Jannick in our Wednesday daily and he said that icons are only used in 2 use cases:

1. interactively inside another component, e.g. a button. In this case the `<button>` is the relevant aria component so it will have a title
2. non-interactively / standalone only as visual component. Since its only visual we can hide it for aria

So for both cases, the icon itself is not relevant for aria

Reply:
Also MDN explains that `aria-hidden` is often used on icons: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden

Reply:
ok, then we just have to remember it once we do the icon-button :)